Project_BookStoreCT/Views/Home/ViewCart.cshtml

```diff
@@ -44,7 +44,7 @@
                         <td class="product-thumbnail"><a href="#"><img src="~/imgs/@cart.image" alt="man" /></a></td>
                         <td class="product-name"><a href="#"><b>@cart.bookname</b></a></td>
                         <td class="product-price"><span class="amount"> @String.Format("{0:n0}", cart.price) USD </span></td>
-                        <td class="product-quantity"><input id="quantity" name="quantity" oninput="javascript: if (this.value.length > this.maxLength) this.value = this.value.slice(0, this.maxLength);" type = "number" maxlength = "3" value="3"></td>
+                        <td class="product-quantity"><input id="quantity" name="quantity" type="number" value="@cart.number"></td>
```

Comment:
<td class="product-quantity"><input id="quantity" name="quantity" oninput="javascript: if (this.value.length > this.maxLength) this.value = this.value.slice(0, this.maxLength);" type = "number" maxlength = "3" value="@cart.number"></td>

Reply:
ừ